pkg/kubelet/kubelet.go

```diff
@@ -454,7 +454,10 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		r := cache.NewReflector(nodeLW, &v1.Node{}, nodeIndexer, 0)
 		go r.Run(wait.NeverStop)
 	}
-	nodeInfo := &predicates.CachedNodeInfo{NodeLister: corelisters.NewNodeLister(nodeIndexer)}
+
+	cacheNodeInfos := make(map[string]corelisters.NodeLister, 1)
```

Comment:
per the current design, each node ONLY belongs to one RP. so kubelet will not need this array here.

Reply:
> per the current design, each node ONLY belongs to one RP. so kubelet will not need this array here.

As we discussed over zoom, this change is due to implementation changes of NewCachedNodeInfo() in scheduler